claims/module-claims/claim-cs3217.js

```diff
@@ -42,34 +42,43 @@ var config = {
   activities_list_fn: function () {
     var activities_list = [];
 
-    for (var week = 1; week <= 2; week++) {
+    const prep_weeks = [1, 3, 5, 7, 9];
+    const ps_weeks = [2, 4, 6, 8, 10];
+    const ps_rates = [1, 1, 1.5, 1.5, 2];   // Hrs per submission
+
+    // ATTENTION: FILL THIS IN
+    const ps_students = [0, 0, 0, 0, 0];    // No. of students per PS
+
+    // Great big hack that might just happen to work without importing a time library
+    function make_time(hrs) {
+      return { start_time: String(1200),
+               end_time: String(1200 + Math.floor(hrs) * 100 + hrs % 1 * 60) };
+    }
+
+    // Preparation claims (2h * 5 = 10h)
+    for (var week of prep_weeks) {
       activities_list.push({
-        activity_type: Claim.COURSE_MATERIAL_PREPARATION,
+        activity_type: Claim.ASSIGNMENT_MARKING,  // Waikay didn't approve "course material preparation"
         week: week,
         day: 'SUNDAY',
-        start_time: '1300',
-        end_time: '1800'
-      });
-    }
+        start_time: '1200',
+        end_time: '1400'
+      });  
+    }    
```

Comment:
/nit these spaces lol

Reply:
Tsk @louietyj never configure editor to trim spaces on save

Reply:
Notepad 😒